zenoh/__init__.py

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2022 ZettaScale Technology
+# Copyright (c) 2024 ZettaScale Technology
```

Comment:
I think we don't need to update the year here.

Reply:
https://github.com/eclipse-zenoh/zenoh-python/pull/184#discussion_r1579621201